Vizzuality/landgriffon
# update base data imported with lg v0.1 methodology

### General description

Update base data imported according to the LandGriffon v0.1 methodology. This update includes:

- material csv to include just mapspam and glw3 data.
- new LG base indicators
- New units and unit conversion

As part of this PR we need to include:

- the update of the h3 commodity ingestion
- the ingestion of the new indicators with the 50km kernel approach (just applied to deforestation, carbon and biodiversity)

### Todo

#### MapSpam

- [x] Remove earthstat pipeline from h3 data ingestion 
- [x] Add mapSpam new comodities (Add list of new datasets @elpamart )

#### New indicators


- [x] Populate indicator coefficients with blue water footprint values by commodity and country ( link spreadsheet by @elpamart ) and link with water use indicator
- [x] Aqueduct data and linked with unsustainable water use
- [x] Deforestation as area with 50 km mean kernel
- [x] Carbon emission with 50 km kernel (net flux layer * deforestation) plus 50km kernel
- [ ] Landuse: update code

### Testing instructions

Check that the seed-source-data pass successfully. 

- Apart from the added feature / bug fix, check overall performance, styling...

## Checklist before merging
